examples/realTimeVideo-config/src/main.ts

```diff
@@ -0,0 +1,94 @@
+import { Authentication, Fleet, KeyValue } from "@formant/data-sdk";
+import "@formant/ui-sdk-joystick";
+import "@formant/ui-sdk-realtime-player";
+import { RealtimePlayer } from "@formant/ui-sdk-realtime-player";
+import "./style.css";
+
+const loadVideo = async (defaultCamera: number) => {
+  (el("formant-realtime-player") as RealtimePlayer).drawer.start();
+  let indicator = document.getElementById("loading-indicator");
+  try {
+    if (await Authentication.waitTilAuthenticated()) {
+      const device = await Fleet.getCurrentDevice();
+
+      await device.startRealtimeConnection();
+      device.addRealtimeListener((_peerId: any, message: any) => {
+        (el("formant-realtime-player") as RealtimePlayer).drawVideoFrame(
+          message.payload.h264VideoFrame
+        );
+      });
+      let videoStreams = await device.getRealtimeVideoStreams();
+      let settingsIcon = document.getElementById("settings");
+      let settingsBlock = document.getElementById("video-settings");
+
+      settingsIcon!.style.display = "flex";
+      settingsIcon?.addEventListener("click", () => {
+        settingsBlock!.style.display = "flex";
+      });
```

Comment:
i'm curious whats up with the Optional Chaining operator (`?`) here on line 25 here? Given that its _already_ asserted to exist with `!` in the previous line. 

Perhaps, a clearer way to perhaps express might look something like:

```ts
// Assert the elements exist during the query…
const settingsIcon = document.getElementById("settings")!;
const settingsBlock = document.getElementById("video-settings")!;

// …then you can just use them.
settingsIcon.style.display = "flex";
settingsIcon.addEventListener("click", () => {
  settingsBlock.style.display = "flex";
});
```